requirements/dev-requirements.txt

```diff
@@ -645,9 +645,7 @@ turn-python==0.0.1
 twilio==6.5.1
     # via -r base-requirements.in
 typing-extensions==4.1.1
-    # via
-    #   black
-    #   django-countries
+    # via django-countries
```

Comment:
Not sure why it made this change. Both locally and in the github actions build, the deleted lines are what I see.